learn/safe-core-account-abstraction-sdk/protocol-kit/README.md

```diff
@@ -1,7 +1,278 @@
+export OWNER_2_PRIVATE_KEY='<PRIVATE_KEY>'
+export OWNER_3_PRIVATE_KEY='<PRIVATE_KEY>'
+```
+
+Install the core SDKs.
+
+```bash
+yarn add @safe-global/safe-core-sdk-types @safe-global/safe-core-sdk @safe-global/safe-ethers-lib
+```
+
+Create an `index.js` file that you will use to run the following code snippets:
+
+```bash
+touch index.ts
+```
+
+### Initialize Signers, Providers, and EthAdapter
+
+The provider is the object that connects to the Ethereum blockchain. The signers are objects that trigger transactions to the Ethereum blockchain or off-chain transactions.
+
+You can get a public RPC URL from [Chainlist](https://chainlist.org), however, public RPC URLs can be unreliable so you can also try a dedicated provider like Infura or Alchemy.
+
+For this tutorial, we will be creating a Safe on the Goerli testnet. 
+
+```tsx
+import { ethers } from 'ethers'
+import EthersAdapter from '@safe-global/safe-ethers-lib'
+
+// <https://chainlist.org/?search=goerli&testnets=true>
+const RPC_URL='<https://eth-goerli.public.blastapi.io>'
+const provider = new ethers.providers.JsonRpcProvider(RPC_URL)
+
+// Initialize signers
+const owner1Signer = provider.getSigner(process.env.OWNER_1_PRIVATE_KEY)
+const owner2Signer = provider.getSigner(process.env.OWNER_2_PRIVATE_KEY)
+const owner3Signer = provider.getSigner(process.env.OWNER_3_PRIVATE_KEY)
+
+const ethAdapterOwner1 = new EthersAdapter({
+  ethers,
+  signerOrProvider: owner1Signer
+})
+```
+
+### Initialize the Safe Service Client
+
+The [Safe Service Client](https://github.com/safe-global/safe-core-sdk/tree/main/packages/safe-service-client) consumes the [Safe Transaction Service API](https://github.com/safe-global/safe-transaction-service). To start using this library, create a new instance of the `SafeServiceClient` class, imported from `@safe-global/safe-service-client`, and pass the URL to the constructor of the Safe Transaction Service you want to use depending on the network.
+
+We will be using Goerli for this tutorial, however, you can also get [service URLs for different networks](https://docs.gnosis-safe.io/learn/infrastructure/available-services).
+
+```tsx
+import SafeServiceClient from '@safe-global/safe-service-client'
+
+const txServiceUrl = 'https://safe-transaction-goerli.safe.global'
+const safeService = new SafeServiceClient({ txServiceUrl, ethAdapter })
+```
+
+### **Initialize the Safe Core SDK**
+
+Goerli is a supported network so we don’t need to specify the contract addresses, however, to see how to create a safe on a local or unsupported network, see [Guide: Integrating the Safe Core SDK](https://github.com/safe-global/safe-core-sdk/blob/main/guides/integrating-the-safe-core-sdk.md#instantiate-an-ethadapter).
+
+Safe Factory is used to create a new Safe. While Safe class represents an instance of a  particular Safe account.
+
+```tsx
+import { SafeFactory } from '@safe-global/safe-core-sdk'
+
+const safeFactory = await SafeFactory.create({ ethAdapterOwner1 })
```

Comment:
```suggestion
const safeFactory = await SafeFactory.create({ ethAdapter: ethAdapterOwner1 })
```

Reply:
This is an outdated version, more recent commits were made here: https://github.com/safe-global/safe-docs/pull/165/commits/2e37808961a0a25f58c5cf252183f3f29d6481f0